rest_server/IOMgrConnection.go

```diff
@@ -0,0 +1,386 @@
+package main
+
+import (
+	"SiodbIomgrProtocol"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"github.com/gin-gonic/gin"
+	"github.com/golang/protobuf/proto"
+)
+
+var (
+	IOMgrChunkMaxSize         uint32 = 65536
+	IOMgrChunkMaxBufferedSize uint64 = 1021
```

Comment:
Actually, it was 1024 (I forgot to switch back after testing different values).
I tried 1024 and 65535. The response time was smaller with 1021 and output "smoother" printing smaller pieces of data.

Reply:
Then I suggest to make this tunable parameter like `rest_server.iomgr_io_buffer_size`, because you tuned this on you machine, by checking only 2 values, on the different machine with different characteristics, result may differ.